contents/julia_101.md

```diff
@@ -2037,6 +2037,43 @@ sym = Symbol(s)
 One simple benefit of symbols is that you have to type one character less, that is, `:some_text` versus `"some text"`.
 We use `Symbol`s a lot in data manipulations with the `DataFrames.jl` package (@sec:dataframes) and data visualizations (@sec:plots and -@sec:makie).
 
+### Splat Operator {#sec:splat}
+
+In Julia we have the "splat" operator `...` which is mainly used in function calls as a **sequence of arguments**.
+We will occasionally use splatting in some function calls in **data manipulation** and **data visualization** chapters.
+
+The `...` operator takes a collection (often an array, vector, tuple or range) and converts into a sequence of arguments.
+The `...` is included after the collection that we want to "splat" into a sequence of arguments.
+For example, if we call a function on a range followed by `...`, it is the same as calling a function on all arguments of a collection.
+Syntactically speaking the following are the same:
+
+```julia
+collection = [x, y, z]
+
+function(collection...) = function(x, y, z)
+```
+
+Anytime Julia sees a splatting operator inside a function call, it will be converted on a sequence of arguments for all elements of the collection separated by commas.
+To understand the `...` operator, let's make a small example with a function that adds all elements in a collection:
+
+```jl
+scob("""
+function add_elements(xs...)
+    return reduce(+, xs)
```

Comment:
Reduce is a very difficult function to understand for non-programmers, I think.

Can we write a, b, c out explicitly?

Reply:
I can do this, see if it is better:

```julia
function add_elements(xs...)
    return +(xs...)
end
```

Reply:
haha but now you have slurping

My suggestion was to make it explicit `add_elements(a, b, c) = a + b + c`. Everyone knows plus as being an infix operator.